00_Base/src/interfaces/router/Authenticator.ts

```diff
@@ -6,7 +6,7 @@ export interface IAuthenticator {
   authenticate(
     allowUnknownChargingStations: boolean,
     identifier: string,
-    username?: string,
```

Comment:
these need to stay optional. security profile 0 exists and may still want to exclude unknown chargers. security profile 3 also exists and does not use a username/password.